com.unity.render-pipelines.high-definition/Editor/VFXGraph/Outputs/VFXLitCubeOutput.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace UnityEditor.VFX
 {
-    [VFXInfo]
```

Comment:
You merged the wrong branch at https://github.com/Unity-Technologies/Graphics/pull/3498/commits/ceba4b6ad69492f1fae2678d5d0057ef9cf1ed63
So, it became a vfx/staging merge.

Reply:
Yes my bad... Hopefully the changes are really minor (only sphere and cube as experimental), so it's fine